server/server.go

```diff
@@ -172,9 +172,24 @@ func (s *server) initOrResetRPCInfoFunc() func(rpcinfo.RPCInfo, net.Addr) rpcinf
 
 func (s *server) buildInvokeChain() {
 	innerHandlerEp := s.invokeHandleEndpoint()
+	// if there is streaming method, make sure the ctxInjectMW is the last middleware
```

Comment:
Temporary fix with comments to clearly explain
// TODO(yuxuan): fix ctx diverge problem temporarily, it should be removed after has new streaming api with ctx

Reply:
done.